linux-anvil/entrypoint

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash -il
 
+# Create conda user with the same uid as the host, so the container can write
+# to mounted volumes
+# Adapted from https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
+USER_ID=${HOST_USER_ID:-9001}
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m conda
+export HOME=/home/conda
+chown -R conda /opt/conda
```

Comment:
Apparently we need the user to re-run the `conda config` stuff (channels enabled and such) for the `conda` user. Alternatively we could just copy `/root/.condarc` to `/home/conda/.condarc` and run `chown` on the copy too.